AppController/djinn.rb

```diff
@@ -4774,6 +4774,9 @@ def prep_app_dashboard()
       setup_app_dir(AppDashboard::APP_NAME, true)
     }
 
+    # Allow fewer dashboard instances for small deployments.
+    min_dashboards = [3, get_all_appengine_nodes.length].min
```

Comment:
In practice, we would be putting a dashboard AppServer on each node (unless we have more than 3 appengine machines): https://github.com/AppScale/appscale/blob/master/AppController/djinn.rb#L5797-L5809